apps/backend/lib/src/auth.ts

```diff
@@ -0,0 +1,77 @@
+import { Request, Response, NextFunction } from "express";
+import * as jwt from "jsonwebtoken";
+import { Algorithm } from "jsonwebtoken";
```

Comment:
Can we clean up this import? Preferabbly just import whats needed from jsonwebtoken